### DISCUSSION TITLE
Error while using react-table[v7] with storybook

### DISCUSSION THREAD

**Discussion by ganeshmani** [TanStack/table#2694] [General]
I am getting this error when i tried to use react-table with storybook.
 
<img width="1215" alt="Screenshot 2020-08-31 at 6 05 29 PM" src="https://user-images.githubusercontent.com/17050715/91720323-8ff41480-ebb4-11ea-91ba-e021ac13b62a.png">

Here's  my webpack config

```
const path = require("path");

module.exports = async ({ config }) => {
  // styles
  config.module.rules.push({
    test: /\.(sass|scss)$/,
    use: ["resolve-url-loader"],
    include: path.resolve(__dirname, "../")
  });
  // fonts
  config.module.rules.push({
    test: /\.(woff|woff2|eot|ttf)$/,
    use: [
      {
        loader: "file-loader",
        query: {
          name: "[name].[ext]"
        }
      }
    ],
    include: path.resolve(__dirname, "../")
  });

  return config;
};
```

**Comment by michaeljota** [comment]
I know it's been a while, but did you manage to solve this? I'm unable to use this with Storybook, even in a new project.
